components/greeting.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [slug, setSlug] = useState('')
+  const hello = useState('It is nice to meet you ')
```

Comment:
This doesn't need to be stored in state it's not going to change. Also `useState` returns an array.